src/pudl/metadata/resources/ferc1.py

```diff
@@ -703,6 +703,28 @@
         "etl_group": "ferc1_disabled",
         "field_namespace": "ferc1",
     },
+    "electric_operating_revenues_ferc1": {
+        "description": "Electric operating revenues - The structed part of schedule 300.",
+        "schema": {
+            "fields": [
+                "utility_id_ferc1",
+                "report_year",
+                "record_id",
+                "revenue_type",
+                "amount",
```

Comment:
if this $'s are always revenues, could this be `revenue` instead? We really should settle on how we name all of these $ columns re: #2179 .. bc we haven't really settled on anything I do think `amount` is fine if you prefer it!

Reply:
I actually prefer `revenue`, and had considered that initially, but went with `amount` to try and be consistent with other tables. I've changed to `revenue` now.